scripts/build.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/sh
+
+ROOT=$(git rev-parse --show-toplevel)
+
+cd ${ROOT}/crates
+make release
```

Comment:
I think it would be better the default is debug. WDYT?

Reply:
I think in general that would be a good idea, but for now, this is only used by the integration_tests, which need the youki binary to be in release mode. As mentioned in the point 1 in the PR comment, I plan to improve scripts and makefiles in a separate PR,  where I will make the default be the debug mode.